src/main/java/life/qbic/expdesign/io/MHCLigandDesignReader.java

```diff
@@ -397,7 +390,9 @@ public Object parse(String value) {
               fillMetadata(header, row, meta, factors, loci, SampleType.Q_MHC_LIGAND_EXTRACT));
           ligandExtract.addProperty("Q_ANTIBODY", antibody);
           String[] mhcClass = getMHCClass(antibody);
-          if (mhcClass.length == 1) {
+          if (mhcClass == null) {
+            return null;
+          } else {
```

Comment:
Could this be also a multitude of 1? If so then you'd need to check that to make sure that that is not the case 👁️

Reply:
good point. this is not important for the use case I am trying to fix, but I need to look into this, it is possible metadata about the MAE antibody (for both MHC classes) is lost, here